students/tihsle/lesson03/mailroom.py

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env python3
+
+import sys
+
+donors = [  ("William Gates, III", [65000.00,10.00]),
+            ("Mark Zuckerberg", [1442.25, 4000.00, 5000.00]),
+            ("Jeff Bezos", [877.33]),
+            ("Paul Allen", [400.00, 200.0000, 10.00]),
+            ("Joe Smithe", [1000.00, 2000.00])
+            ]
+
+
+def send_thank_you():
+    #send a thank you
+
+    #which donor
+    donor = select_donor()
+
+    #how much is the gift
+    get_donation(donor)
```

Comment:
the name of the function doesn't match to what the function is doing - this should be `set_donation`